src/Ninja/G1CP/Content/Fixes/Session/fix225_MordragNcGetOut.d

```diff
@@ -0,0 +1,48 @@
+/*
+ * #225 Mordrag can be told to "get out" in New Camp
+ */
+func int G1CP_225_MordragNcGetOut() {
+    const string CONDITION = ""; CONDITION = "Org_826_Mordrag_HauAb_Condition";
+
+    if (!G1CP_IsFunc(CONDITION, "int|none"))
+    || (!G1CP_IsInfoInst("Org_826_Mordrag_GotoNewcamp"))
+    || (!G1CP_IsIntVar("Thorus_MordragKo", 0)) {
+        return FALSE;
+    };
+
+    HookDaedalusFuncS(CONDITION, "G1CP_225_MordragNcGetOut_Hook");
+    return TRUE;
+};
+
+/*
+ * This function intercepts the dialog condition to introduce more conditions
+ */
+func int G1CP_225_MordragNcGetOut_Hook() {
+    G1CP_ReportFuncToSpy();
+
+    // Define possibly missing symbols locally
+    const int LOG_RUNNING = 1;
+
+    // Add the new conditions (other conditions remain untouched)
+    var int cond1;
+    var int cond2;
+
+    // Check if dialog was told (check if symbol exists first!)
+    var int symbId; symbId = MEM_GetSymbolIndex("Org_826_Mordrag_GotoNewcamp");
+    if (symbId != -1) {
+        cond1 = !Npc_KnowsInfo(hero, symbId);
+    } else {
+        cond1 = TRUE; // If the dialog does not even exist take no chances
+    };
```

Comment:
This construct is not necessary (may have been some left over in a similar fix). The existence of "Org_826_Mordrag_GotoNewcamp" is already guaranteed by the function above. If it does not exist, this function here would never be executed. This one line here should suffice:
```D
// Check if dialog was told
cond1 = (!Npc_KnowsInfo(hero, MEM_GetSymbolIndex("Org_826_Mordrag_GotoNewcamp")));
```